models/core/src/index.ts

```diff
@@ -43,4 +43,12 @@ export function createModel (builder: Builder): void {
     TReference,
     TShortRef
   )
+
+  builder.createDoc(
```

Comment:
I don't think we need it in model

Reply:
I think if we have system user account that it need be create